manifests/params.pp

```diff
@@ -25,7 +25,13 @@
     }
     'RedHat': {
       $package          = 'tftp-server'
-      $daemon           = false
+      if $::operatingsystemrelease =~ /^7/ {
```

Comment:
Maybe EL6 is the exception here? I'd expect Fedora to have the tftp.socket service as well.